### ISSUE TITLE
Can you change swin-transformer as backbone?

### ISSUE THREAD

**Issue by launchauto** [facebookresearch/dino#82]
Dear author, I retry your code and get success.Then I just replace the backbone with swin-transformer and test the linear evaluation, during the training the loss decays from 8.01 to 2.05. The model result is very bad-----the top1-accuracy is nearly 0 in the first epoch of fine-tuning. I don't change any hyperparameters, what is wrong with me?

**Comment by amobiny** [comment]
Were you able to find the answer?

**Comment by rgbd-zml** [comment]
> 
Did you solve this problem? 
what is the top1-accuracy after you replace the backbone?

**Comment by kirill-menke** [comment]
Yes, you can do that by specifying the `--arch` option, e.g. `python main_dino.py --arch swin_b`.
However, you need to change the following line
https://github.com/facebookresearch/dino/blob/cb711401860da580817918b9167ed73e3eef3dcf/main_dino.py#L178
to `embed_dim = student.head.weight.shape[1]` since the final layer is named `head` in the Swin-Transformer models.
